frontend/src/components/professor/GameSummary/GameSummary.js

```diff
@@ -53,7 +53,7 @@ function GameSummary(props) {
   ]
 
   const activityStatsCardTitles = [
-    { text: 'Liczba ocenionych aktywności:', value: summaryDetails?.assessedActivitiesCounter },
+    { text: 'Liczba ocenionych aktywności:', value: summaryDetails?.assessedActivityCounter },
     { text: 'Liczba nieocenionych aktywności:', value: summaryDetails?.notAssessedActivityCounter },
     { text: 'Liczba odpowiedzi oczekujących na sprawdzenie:', value: summaryDetails?.waitingAnswersNumber }
```

Comment:
Tutaj trzeba będzie zweryfikować z backiem czy te wartości się zgadzają - po ocenie jednej z nich podbiło mi o 1 `waitingAnswersNumber` i zredukowało o 1 `assessedActivityCounter` - ale to (raczej) poza scope tego PR

Reply:
cała ta karta ma bugi, wszystkie wartości w niej są błędne, na razie nie zwracaj uwagi, @barto14753 już nad tym pracuje